integrations/stripe/subscriptions/client/react/README.md

```diff
@@ -0,0 +1,23 @@
+## Diagonal with Stripe integration client example
+
+This example shows how you can build a subscription integration with Stripe using [React](https://reactjs.org/).
```

Comment:
I think it would be nice to clarify this in the first readme, that these 2 folders are the client & server. Also, this and the server one can be removed?